shared/search/user-search/render.native.js

```diff
@@ -0,0 +1,123 @@
+// @flow
+import React, {Component} from 'react'
+import {Avatar, Box, Icon, Input, Text, ListItem} from '../../common-adapters'
+import {globalStyles, globalColors} from '../../styles/style-guide'
+
+import type {Props, SearchResult} from './render'
+import type {Combo} from '../../constants/types/more'
+import type {Props as TextProps} from '../../common-adapters/text'
+
+function EmboldenTextMatch ({text, match, style, textType, emboldenStyle}: {text: string, match: string, emboldenStyle: Object, style: Object, textType: TextProps.type}) {
+  const indexOfMatch = text.indexOf(match)
```

Comment:
In the mockup, "Malg" in "John Malg" is bolded. Should this match be case-insensitive?